modules/hcp-bm-hc-console.adoc

```diff
@@ -28,7 +28,7 @@ The **Create cluster** page is displayed.
 ====
 As you enter details about the cluster, you might find the following tips useful:
 
-** If you want to use predefined values to automatically populate fields in the console, you can create a host inventory credential. For more information, see _Creating a credential for an on-premises environment_.
+** If you want to use predefined values to automatically populate fields in the console, you can create a host inventory credential. For more information, see _Creating a credential for an on-premise environment_.
```

Comment:
I think this should stay as "on-premises" because it's referencing a title that uses that term.